dataland-frontend/src/components/pages/SearchData.vue

```diff
@@ -1,25 +1,34 @@
 <template>
-  <TheHeader/>
-  <TheContent>
-  <div class="grid">
-    <div class="col-12">
+  <AuthenticationWrapper>
+    <TheHeader/>
+    <TheContent>
       <div class="grid">
-        <RetrieveCompany/>
-        <RetrieveSkyminder/>
+        <div class="col-12">
+          <div class="grid">
+            <RetrieveCompany/>
+            <RetrieveSkyminder/>
+          </div>
+        </div>
       </div>
-    </div>
-  </div>
-  </TheContent>
+    </TheContent>
+  </AuthenticationWrapper>
 </template>
 
 <script>
 import RetrieveSkyminder from "@/components/forms/RetrieveSkyminder";
 import RetrieveCompany from "@/components/forms/RetrieveCompany";
 import TheHeader from "@/components/structure/TheHeader";
 import TheContent from "@/components/structure/TheContent";
+import AuthenticationWrapper from "@/components/wrapper/AuthenticationWrapper";
```

Comment:
Muss die Komponente `AuthenticationWrapper` immer die "abgesicherten" Komponenten umschließen?

Reply:
ja genau, in diesem wrapper findet der check statt ob jemand eingeloggt ist, falls nicht wird auf die Startseite umgeleitet